### PR TITLE
doc: add the v1.2.0-patch1 ISO for the usb installation issue

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
related to https://github.com/harvester/harvester/issues/4510

Add the information of patched ISO

### CODE REVIEW

File: docs/install/usb-install.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ There are a couple of ways to create a USB installation flash drive.
 :::caution
 **Known Issue:** For the v1.2.0 ISO image, there is a known [issue](https://github.com/harvester/harvester/issues/4510) where the interactive ISO installation will get stuck using the USB method.
 
+Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack v1.2.0 ISO with the correct lable, and there is no other change. Also, there is a sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) for verify the ISO.
```
Comment (vickyhella) [inline docs/install/usb-install.md:18]:
```suggestion
Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack v1.2.0 ISO with the correct lable, and there is no other change. Also, you can use this sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.
```

Comment (Vicente-Cheng) [reply docs/install/usb-install.md]: Hi @vickyhella, thanks for the review!
Updated

File: docs/install/usb-install.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ There are a couple of ways to create a USB installation flash drive.
 :::caution
 **Known Issue:** For the v1.2.0 ISO image, there is a known [issue](https://github.com/harvester/harvester/issues/4510) where the interactive ISO installation will get stuck using the USB method.
 
+Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack v1.2.0 ISO with the correct lable, and there is no other change. Also, you can use this sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.
```
Comment (vickyhella) [inline docs/install/usb-install.md:18]:
```suggestion
Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack the v1.2.0 ISO with the correct label and nothing else is changed. Also, you can use this sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.
```

Comment (vickyhella) [reply docs/install/usb-install.md:18]: One more suggestion... (sorry for missing this one earlier)

Comment (Vicente-Cheng) [reply docs/install/usb-install.md]: Updated, thanks!

**Review by vickyhella — APPROVED** [review]

File: docs/install/usb-install.md [hunk 3]
```diff
@@ -15,6 +15,8 @@ There are a couple of ways to create a USB installation flash drive.
 :::caution
 **Known Issue:** For the v1.2.0 ISO image, there is a known [issue](https://github.com/harvester/harvester/issues/4510) where the interactive ISO installation will get stuck using the USB method.
 
+Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack the v1.2.0 ISO with the correct label and nothing else is changed. Also, you can use this sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.
```
Comment (guangbochen) [inline docs/install/usb-install.md:18]:
```suggestion
To resolve this, you can use the patched [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso). This patched version only corrects the label; no other changes have been made. You can also use the related sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.
```

**Review by guangbochen — APPROVED** [review]

**Review by bk201 — APPROVED** [review]

**Pull request merged**